opensrp-reveal/src/main/res/values/colors.xml

```diff
@@ -17,7 +17,7 @@
     <color name="unsprayable">#000000</color>
     <color name="sprayed">#6CBF0F</color>
     <color name="unsprayed">#EE0427</color>
-    <color name="partially_sprayed">#ED8231</color>
+    <color name="partially_sprayed">#6CBF0F</color>
```

Comment:
I changed this color to green in the case we have old forms that are played, it will show up as green.